OmniKit/MessageTransport/MessageBlocks/PodInfoFaultEvent.swift

```diff
@@ -16,13 +16,13 @@ public struct PodInfoFaultEvent : PodInfo, Equatable {
     public let deliveryStatus: DeliveryStatus
     public let insulinNotDelivered: Double
     public let podMessageCounter: UInt8
-    public let unknownPageCode: Data
-    public let previousStatus: FaultEventCode
+    public let totalInsulinDelivered: Double
     public let currentStatus: FaultEventCode
     public let faultEventTimeSinceActivation: TimeInterval?
-    public let reservoirLevel: Double?
+    public let reservoirLevel: String
```

Comment:
The internal rep of this should stay `Double?`.  Conversion to string form should be done at display time.

Reply:
reverted back to Double? and formatted at display time.